graylog-storage-elasticsearch7/src/test/java/org/graylog/storage/elasticsearch7/views/export/ElasticsearchExportBackendIT.java

```diff
@@ -64,6 +78,11 @@ public void setUp() {
 
     }
 
+    @After
+    public void afterEach() {
+        elasticsearch.cleanUp();
+    }
+
```

Comment:
As we test exports only, which do not change indexed docs at all, should we in fact waste time importing fixtures before each test and cleaning them after each tests?
Wouldn't it be more efficient to load everything once and distinguish different test cases by different indices names (`graylog_*` in one set of tests, `graylog2_*` in another?)